app/jobs/submit_folio_request_job.rb

```diff
@@ -18,7 +18,7 @@ def perform(request_id, options = {})
 
     logger.debug("FOLIO response: #{response}")
     request.merge_ils_response_data(FolioResponse.new(response.with_indifferent_access))
-    request.save
+    request.save!
```

Comment:
I thought `save!` raises exceptions?

Reply:
Sorry, please reload. I forgot to save the file before committing.